proposals/Symbiosis Protocol.md

```diff
@@ -0,0 +1,387 @@
+
+Better yet unlike all these other web apps inter planetary ones can access each others documents all it takes, you've guessed it granting a permission!
+
+-->
+
+##### Impact
+_How directly important is the outcome to our top-level mission?_
+
+<!--
+Explain why you have chosen this rating
+What awesome potential impact/outcomes/results will we see if we nail this project?
+-->
+
+🔥🔥🔥 
+
+<!-- - Addresses some of the long standing problems (outlined in appendix) that teams building on IPFS are experiencing and are forced to work around. -->
+
+- Enables teams to plug domain specific extensions without having to fork our stack.
+- Enables IPFS nodes to leverage local infrastructure.
+- Enables short-lived IPFS node use cases (e.g. mobile devices, serverless computing).
+
+
+##### Leverage
+_How much would nailing this project improve our knowledge and ability to execute future projects?_
+
+
+<!-- Explain the opportunity or leverage point for our subsequent velocity/impact (e.g. by speeding up development, enabling more contributors, etc)
+-->
+
+🎯🎯🎯
+
+- Enables IPFS to expand into areas like [serverless computing][cloud computing] where both full IPFS node and IPFS Client are impractical.
+- Lays foundation for **multi tenanant** nodes that are needed to realize full potential of embeded IPFS  in web browsers (e.g brave) and/or OS.
+- By enabling domain specific protocol extensions we grow the network instead of fragmenting it.
+
+##### Confidence
+_How sure are we that this impact would be realized? Label from [this scale](https://medium.com/@nimay/inside-product-introduction-to-feature-priority-using-ice-impact-confidence-ease-and-gist-5180434e5b15)_.
+
+<!--Explain why this rating-->
+
+Level 3
+
+There is enough evidence supported by our own [ad-hock stop-gap solutions](#Ad-hoc-stop-gap-solutions) combined with [solutions our  collaborators need to resort for](#Unintentionally-Incompatible-infrastructure) to justify this level of confidence.
+
+### Project definition
+##### Brief plan of attack
+
+<!--Briefly describe the milestones/steps/work needed for this project-->
+
+1. Specificy [symbiosis wire protocol](#Symbiosys-protocol).
+2. Implement domain service that host announces (will figure out the right one by talking to collabs).
+3. Implement permission handling flow on the host.
+4. Make symbiont leverage announced service through the protocol.
+5. Collaborate with teams to validate design with differnt use cases.
+6. Take learnings and implement zeroconfig service discovery.
+
+##### What does done look like?
+_What specific deliverables should completed to consider this project done?_
+
+- Have at least one collab adopt symbiosis protocol.
+- IPFS Desktop providing services that enhances in-browser node capabilities.
```

Comment:
Is this possible?

Reply:
Yes, if implemented. But question suggest maybe we need to elaborate more on this ?